command/connect/envoy/envoy_test.go

```diff
@@ -835,32 +859,8 @@ func TestGenerateConfig(t *testing.T) {
 				ProxySourceService: "",
 				GRPC: GRPC{
 					AgentAddress: "127.0.0.1",
-					AgentPort:    "8502",
-				},
-				AdminAccessLogPath:    "/dev/null",
-				AdminBindAddress:      "127.0.0.1",
-				AdminBindPort:         "19000",
-				LocalAgentClusterName: xds.LocalAgentClusterName,
-				PrometheusScrapePath:  "/metrics",
-			},
-		},
-		{
-			Name:  "CONSUL_HTTP_ADDR-with-https-scheme-does-not-affect-grpc-tls",
-			Flags: []string{"-proxy-id", "test-proxy", "-ca-file", "../../../test/ca/root.cer"},
-			Env:   []string{"CONSUL_HTTP_ADDR=https://127.0.0.1:8500"},
-			WantArgs: BootstrapTplArgs{
```

Comment:
This diff looks bizzare because I removed an unnecessary testcase and renamed some tests at the same time.